README.md

```diff
@@ -1 +1,143 @@
-# blog
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Blog app ](#-blog-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Run](#run)
+    - [Test](#test)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Blog app <a name="about-project"></a>
+
+**Blog app** will be a classic example of a blog website. I will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts. Everything will be based on the given UML class diagram. The data will be stored in postgreSQL database. 
+
+The project is divided in eleven (11) milestones and one exercise.
+
+- Milestone 1: Setup and controllers
+- Milestone 2: Controllers specs
+- Milestone 3: Creating a data model
+- Milestone 4: Processing data in models
+- Milestone 5: Views
+- Milestone 6: Forms
+- Milestone 7: Validations, Model specs, and n+1 problems
+- Milestone 8: Add Devise
+- Milestone 9: Add authorization rules
+- Milestone 10: Integration specs for views
+- Milestone 11: Add API endpoints
+- Exercise: API documentation
+  
+
+<p align="center">
+  <img src="./images/blog_user_all_posts.png" alt="All posts by a given user page wireframe"  width="250px"  />
+  <img src="./images/blog_single_post.png" alt="Single post page wireframe"  width="250px"  />
+</p>
+
```

Comment:
![image](https://github.com/Fahim-Khan-P/blog/assets/93039359/3feea621-8d2b-4a93-97b0-53048bf69dbd)

- [ ] It would be nice if your picture links should be visible in your readme file because right now both links are broken 😨 Kindly fix both links to make your readme descriptive and professional 👍